### PR TITLE
feat/segmentation min cut

### PR DESCRIPTION
**Pull request by lengoanhcat**
##### Pre-Merge Checklist

- [x] Tests and/or benchmarks are included
- [ ] Documentation is changed or added
- [x] Lint process passed (`npm run lint`)
- [x] Tests passed (`npm run test`)

### CODE REVIEW

**Comment by luoxuhai** [comment]
@lengoanhcat Need to run `npm run build:wasm` in the local, and then push the code.

**Comment by luoxuhai** [comment]
@lengoanhcat Need to run `npm run build:wasm` in the local, and then push the code.

**Comment by lengoanhcat** [comment]
@luoxuhai : done ! I have to make some extra changes besides binding codes to successfully run .devcontainer and build pcl-core.wasm. Pls help review and let me know it works in your environment or not.

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -69,10 +69,11 @@ RUN echo "## Aggressive optimization: Remove debug symbols" \
...
+     && apt-get install -y python3-pip \
+     && pip3 install --upgrade pip \
+     && pip3 install cmake
 
-RUN npm install --global yarn \
-    && git submodule update --init --recursive --remote \
-    && git config --global --add --bool push.autoSetupRemote true \
-    && yarn setup
\ No newline at end of file
+# RUN npm install --global yarn \
```
Comment (luoxuhai) [inline .devcontainer/Dockerfile:76]: Comments can be deleted :-)

File: src/modules/segmentation/MinCutSegmentation.ts [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+  constructor(PT: TPointTypesUnion = PointXYZ) {
+    this._native = new __PCLCore__[`MinCutSegmentation${PT.name}`]();
+  }
+
+  public setInputCloud(cloud: PointCloud<T>) {
+    return this._native.setInputCloud(cloud._native);
+  }
+
+  public setSigma(sigma: number): void {
```
Comment (luoxuhai) [inline src/modules/segmentation/MinCutSegmentation.ts:23]: `void` and `return` can be removed :-)

File: src/modules/segmentation/MinCutSegmentation.ts [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    return this._native.setSigma(sigma) as void;
+  }
+
+  public getSigma(): number {
+    return this._native.getSigma() as number;
```
Comment (luoxuhai) [inline src/modules/segmentation/MinCutSegmentation.ts:28]: `number` can be removed :-)

**Review by luoxuhai — COMMENTED** [review]
Thanks, I commented on affected line with suggestion :-)

**Comment by luoxuhai** [comment]
Please run `git rebase master(the latest)` locally and resubmit, then I can merge

**Comment by lengoanhcat** [comment]
@luoxuhai done ^^ !

**Comment by luoxuhai** [comment]
Thanks! :-)

**Pull request merged**